wd_capabilities.au3

```diff
@@ -354,84 +335,51 @@ EndFunc   ;==>_WD_CapabilitiesDefine
 Func __WD_CapabilitiesInitialize($s_MatchType, $s_BrowserName = '')
 	Local Const $sFuncName = "__WD_CapabilitiesInitialize"
 	#Region - parameters validation
-	Local $s_SpecificOptions_KeyName = ''
-
-	If $s_BrowserName <> '' Then
+	Local $bAlwaysMatch_exist = (IsObj($_WD_CAPS__OBJECT) And Json_ObjExists($_WD_CAPS__OBJECT, 'capabilities.alwaysMatch'))
+	Local $iFirstMatch_count = UBound(Json_Get($_WD_CAPS__OBJECT, '[capabilities][firstMatch]'))
+	SetError(0) ; for any case because UBound() can set @error
+	__WD_ConsoleWrite($sFuncName & ': #' & @ScriptLineNumber & '   $bAlwaysMatch_exist = ' & $bAlwaysMatch_exist & '  $iFirstMatch_count = ' & $iFirstMatch_count, $_WD_DEBUG_Full)
+
+	If $s_MatchType <> 'alwaysMatch' And Not $bAlwaysMatch_exist Then
+		Return SetError(__WD_Error($sFuncName, $_WD_ERROR_GeneralError, '"alwaysMatch" must be set at first place'))
+	ElseIf $s_MatchType = 'alwaysMatch' And $bAlwaysMatch_exist Then
+		Return SetError(__WD_Error($sFuncName, $_WD_ERROR_AlreadyDefined, '"alwaysMatch" must be set only once'))
```

Comment:
Already defined is uased here.
But also should used in _WD_CapabilitiesDefine()

Reply:
nevermind